### ISSUE TITLE
Recently Completed view is not loading

### ISSUE THREAD

**Issue by cheesi** [hanneswidrig/azure-devops-pull-request-center#25]
The recently completed view is not loading. (it hasn't been working for quite a while)
There are many requests in the background:
![image](https://user-images.githubusercontent.com/5590254/169022616-8bc86d98-4c5e-4b25-8536-efec876ee59d.png)

Sometimes so much, that ADO will limit the API access:
![image](https://user-images.githubusercontent.com/5590254/169022780-149ab159-e694-467f-8646-1f616d3c18c5.png)

Looks like it could be connected to https://github.com/hanneswidrig/azure-devops-pull-request-center/issues/23

The other views (Active, Draft) work without any problems.

Console output
```
root/restoreSettings/fulfilled: completed successfully [store.ts:187:14](https://hannes-widrig.gallerycdn.vsassets.io/extensions/hannes-widrig/pull-request-center/2.2.3/1642443965956/build/static/js/state/store.ts)
root/getPullRequests/fulfilled: completed successfully [store.ts:106:14](https://hannes-widrig.gallerycdn.vsassets.io/extensions/hannes-widrig/pull-request-center/2.2.3/1642443965956/build/static/js/state/store.ts)
```

**Comment by hanneswidrig** [comment]
Are you still having these issues?

**Comment by cheesi** [comment]
Hi!

Yes I can confirm, we are still having this issue.

**Comment by hanneswidrig** [comment]
What's the date range for PRs that you're requesting, is it all time?

**Comment by cheesi** [comment]
Nope, usually 1 month, but it does not matter, I also get it for 14 or 7 days.

**Comment by nhuethmayr** [comment]
Same here, view is not loading - I believe its running into some sort of loop
![image](https://github.com/hanneswidrig/azure-devops-pull-request-center/assets/26460692/32355d5d-9f21-41bc-a057-15b1c05ff405)

**Comment by hanneswidrig** [comment]
@nhuethmayr I think the infinite loop will be easy to resolve, I'll get it fixed this week.

**Comment by nhuethmayr** [comment]
Maybe you can fix #26 as well 😉 That way we can update the extension and get the benefit of both in one go.

**Comment by hanneswidrig** [comment]
> any updates on a PR since the last visit (bold title + blue line on the left side)

This one is definitely possible, and would be time-consuming.

> any tags on the PR

If tags are easily available, this would be possible and hopefully not too much work.

**Comment by hanneswidrig** [comment]
@nhuethmayr The infinite loop should be fixed now in **3.1.0**. Please let me know if you're still experiencing issues.
